colormap.py

```diff
@@ -313,6 +313,7 @@
         ('kaust', 'Brown'),
         #( Extras; these are really randomly found user names; needed for screen)
         #( clarity while in color more)
+        ('j', 'Yellow'),       ## yes, let's start editing the colormap file around here ;-)
```

Comment:
One issue I noticed is that people do not understand the ability to tune colors;
by selecting a random relatively common letter, chance has it that it will trigger some follow up.